Portfolio development

CalebBaines1 opened the pull request:
Personalize portfolio in 3 commits by:

- Updating index.html with personal details

- Adding picture of myself

- Changing the addRandomGreeting function to addRandomFact

CalebBaines1 committed:
Update index.html with personal details.

CalebBaines1 committed:
Add picture of myself.

CalebBaines1 committed:
Change addRandomGreeting functionality to addRandomFact.

blakewilliams1 commented on portfolio/src/main/webapp/index.html:
```diff
@@ -9,6 +9,9 @@
       <h1>Caleb Baines Portfolio</h1>
+      <img src="./images/caleb_pic.jpg" alt="Picture of Caleb"
+      width="200" height="225"
+      />
```
Feel free to condense these 3 lines down to one, it looks like there's enough space

ruixia replied:
+1

CalebBaines1 replied:
```suggestion
     <img src="./images/caleb_pic.jpg" alt="Picture of Caleb" width="200" height="225" />
```

blakewilliams1 committed:
Changing indent of facts array

Just a little easier to read.

I'll have to try and track down some blue moon ice cream somewhere, it looks like a midwest thing :)

blakewilliams1 commented:
Lgtm. I also tried to suggest an indenting edit but but feel free to take it or leave it.

blakewilliams1 reviewed (approved):
lgtm

CalebBaines1 reviewed (commented):
Thanks, I agree.

Blue moon is definitely Midwestern and one of the few flavors I feel does not need toppings.

CalebBaines1 reviewed (commented):
Just barely 2 lines but I think that works.
